src/ICeWall/test/ICeWall.wordlist

```diff
@@ -1,4 +1,3 @@
-centres
```

Comment:
if switching from centre to center, it probably makes sense to include those on this list

Reply:
I didn't add because `center` is the regular spelling in EN-US. The only place the wordlist file is used is for spell checking. My reasoning was that `centre` was added because it is EN-UK.

Reply:
I think @Colin-Holehouse is using that list as a spell checking list, so it's just a custom list of words.
I think you can ignore both my comments, since they relate to the modifying the options and not just the messages.

Reply:
It's a valid concern, I double checked and wirebond was being used both capitalized and not, so I changed the occurrences to be the same on 1562b7c and also updated the doc for the `center` change on 13b6443.

Reply:
This is a good point- the intention was to only change messages for consistency-not the option names or case. @vvbandeira do we need to check this for other tools as well?

Reply:
@dralabeing I don't think we need to do the same for the other tools. This was a special case, since the spelling was correct (although in the UK variant). Changing the message and keeping the option would lead to confusion IMO.

Reply:
ok-good to know. Agreed.